src/renderer/App.vue

```diff
@@ -82,7 +82,30 @@ export default {
           console.log(error)
         }
       })
+    },
+
+    parseCSV(csv) {
```

Comment:
@umutbozdag we used papa parce csv. So we don't need this now. Can you exclude this change from PR?